tests/test_models/test_engine/test_file_storage.py

```diff
@@ -5,7 +5,7 @@
 
 from datetime import datetime
 import inspect
-import models
+from models import storage, storage_t
 from models.engine import file_storage
```

Comment:
Should probably be turned into `from models.engine.file_storage import FileStorage`.

Reply:
Got that one fixed too